### ISSUE TITLE
Allow Supplying a filesystem map `fsMap` via `.executeWithFs({ ... })`

### ISSUE THREAD

**Issue by seveibar** [tscircuit/eval#11]
This will allow us to supply relative imports, for example you might supply:

```tsx
evalWebWorker.executeWithFs({
  fsMap: {
    "main.tsx": `import manualEdits from "./manual-edits.json"\n//....`,
    "manual-edits.json": "{ ... }"
  },
  entrypoint: "main.tsx"
})
```

/bounty $20

**Comment by aybanda** [comment]
/attempt #11

<div id="algora-attempt" />

| [Algora profile](https://console.algora.io/@/aybanda) | Completed bounties | Tech | Active attempts | Options |
| --- | --- | --- | --- | --- |
| @aybanda | 1 bounty from 1 project | <div align="center"></div> | <div align="center"></div> | [Cancel attempt](https://console.algora.io/api/bounties/cm4i69akb000gl603ptsors6j/cancel-attempt) |

**Comment by seveibar** [comment]
fixed in #13
